src/Template/Bake/tests/test_case.ctp

```diff
@@ -89,6 +89,7 @@ class <%= $className %>Test extends TestCase
     public function tearDown()
     {
         unset($this-><%= $subject %>);
+        TableRegistry::clear();
```

Comment:
It's included already because the table which the test is baked against is loaded using the TableRegistry, which is why this 'bug' exists.